packages/package-requester/src/packageRequester.ts

```diff
@@ -386,7 +389,7 @@ function fetchToStore (
       ) {
         let integrity
         try {
-          integrity = await loadJsonFile<Record<string, { size: number, mode: number, integrity: string }>>(path.join(target, 'integrity.json'))
+          integrity = await loadJsonFile<Record<string, { size: number, mode: number, integrity: string }>>(pkgIndexFilePath)
```

Comment:
Having such a complex type inline makes it quite difficult to read. I'd like to suggest defining a type separately.

Reply:
ok, I will do that in a separate commit. Otherwise there'd be too many unrelated changes in this PR.

Reply:
I will name it `PackageIndex`